### PR TITLE
Add 3 New Projects

### PR DESCRIPTION
**Pull request by scottstraughan**
This MR adds three new projects found on GitHub.com.

### CODE REVIEW

File: content/projects/2024/2024-12-16-parallel-computing-in-practice.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - openmp
+  - parallel-computing
+  - openapi
+  - sycl
+  - levelzero
+title: 'Parallel Computing in Practice'
+external_url: https://github.com/apinge/parallel_computing_in_practice
+---
+
+Examples of deploying and computing with SYCL, OpenCL, Level Zero nd OpenMP on GPU devices for enhanced parallel
```
Comment (carlewis) [inline content/projects/2024/2024-12-16-parallel-computing-in-practice.md:18]:
```suggestion
Examples of deploying and computing with SYCL, OpenCL, Level Zero and OpenMP on GPU devices for enhanced parallel 
```

**Review by carlewis — DISMISSED** [review]

**Comment by scottstraughan** [comment]
Thanks @carlewis ! Good typo find 👍 .

**Comment by scottstraughan** [comment]
@carlewis You able to check this?

**Review by carlewis — DISMISSED** [review]

**Comment by carlewis** [comment]
> @carlewis You able to check this again please?

No problem, approved

**Review by codeplaymax — APPROVED** [review]

**Review by codeplaymax — APPROVED** [review]

**Review by codeplaymax — APPROVED** [review]

**Pull request merged**